docs/source/launch_parameters.md

```diff
@@ -0,0 +1,74 @@
+# Launch Parameters for Text Generation Inference
+
+Text Generation Inference allows you to customize the way you serve your models. You can use the following parameters to configure your server. You can enable them by adding them environment variables or by providing them as arguments when running `text-generation-launcher`. Environment variables are in `UPPER_CASE` and arguments are in `lower_case`.
+
+## Model
+
+- `MODEL_ID` - The name of the model to load. Can be a MODEL_ID as listed on huggingface.co like `gpt2` or `OpenAssistant/oasst-sft-1-pythia-12b`. Or it can be a local directory containing the necessary files as saved by `save_pretrained(...)` methods of transformers. Default: `bigscience/bloom-560m`.
```

Comment:
```suggestion
- `MODEL_ID` - The identifier or path of the model to load. Can be a MODEL_ID as listed on huggingface.co like `gpt2` or `OpenAssistant/oasst-sft-1-pythia-12b`, or it can be a local directory containing the necessary files as saved by `save_pretrained(...)` methods of transformers. Defaults to `bigscience/bloom-560m`.
```